steps/pagerduty/0.1.1/step.yml

```diff
@@ -0,0 +1,98 @@
+title: PagerDuty
```

Comment:
Can you maybe set the title to `Create a PagerDuty alert`? It would be more descriptive of the specific action it is doing.